frontend/src/Student/pages/Submit.js

```diff
@@ -1,38 +1,25 @@
 import React from 'react'
 import { Route, Switch } from 'react-router-dom'
 import { Container } from 'reactstrap'
-import { connect } from 'react-redux'
 
 import SubmissionFormChooser from '../components/SubmissionFormChooser'
 import PhotoSubmissionForm from '../containers/PhotoSubmissionForm'
 import VideoSubmissionForm from '../containers/VideoSubmissionForm'
 import OtherMediaSubmissionForm from '../containers/OtherMediaSubmissionForm'
 
-const Submit = props => (
+const addIdPropFromQueryParams = () => {
+  return window.location.search.split('=')[1]
```

Comment:
This is not the most robust way to do this, as any additional query parameters will cause issues.

eg. `/submit?foo=bar&to=1` causes the buttons to link to `/submit/<type>?to=bar`

Additionally, no check is done to make sure a show w/ the id exists, so you can view the chooser for any id and when you attempt to view the submission forms for that non-existent show, the app crashes.

Ideally, we would parse the query string specifically for the param `to` and  check if the id in the `to` query param is a valid show which this student can submit to, otherwise, render a Not Found page.

For MVP purposes, this'll work, but I'm making a story for having more robust query string parsing.